### ISSUE TITLE
cpplint: Erroneously warns about accessors in structs

### ISSUE THREAD

**Issue by alexhenning** [ros/roslint#32]
Accessors for structs should be indented the same as they would in classes (not at all). However, cpplint warns that they should be indented one space. This can likely be updated the same way it was for classes.

Given:
```C++
// Copyright 2015 Alex Henning

struct A
{
public:
}

```

Result:
```
structs.cpp:5:  public: should be indented +1 space inside struct A  [whitespace/indent] [3]
Done processing structs.cpp
Total errors found: 1
```
